en/docs/choreo-concepts/ci-cd.md

```diff
@@ -33,44 +33,42 @@ Choreo can replicate builds from an identical code version (Git commit). This me
 
 ### Trigger a build
 
-On the **Build** page, click **Build Latest**. If necessary you have the option to select a particular commit and build an image. 
+On the **Build** page, click **Build Latest**. If necessary, you have the option to select a particular commit and build an image.
 
-If you want to automatically trigger a build with each commit, you can enable **Auto Build on Commit**. 
+If you want to automatically trigger a build with each commit, you can enable **Auto Build on Commit**.
 
 ### Build logs
 
 You can view build logs for specific builds on the **Build** page.
- 
+
 To view details of a specific build, click **View Details** corresponding to the build.
 
 ## Deployment
 
-Once you build an image in Choreo, you can deploy it via the **Deploy** page. To deploy an image, you can follow one of the approaches given below: 
-
-- Manually deploy: In the **Deploy** page, go to the **Set Up** card and click **Deploy**.
+Once you build an image in Choreo, you can deploy it via the **Deploy** page. To deploy an image, you can follow one of the approaches given below:
 
-- Automatically deploy on build: In the **Deploy** page, go to **Set Up** card and enable **Auto Deploy on Build**. This automatically initiates deployment upon the completion of an automatic build. 
+- **Manually Deploy**: In the **Deploy** page, go to the **Set Up** card and click **Deploy**.
+- **Automatically Deploy on Build**: In the **Deploy** page, go to the **Set Up** card and enable **Auto Deploy on Build**. This automatically initiates deployment upon the completion of an automatic build.
```

Comment:
Same concern as above. There is no specific reason to have "Automatically Deploy on Build" in title caps right?